<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-QBC6F12R" ignored="0" name="Test" tests="3" failures="0" timestamp="12 Jul 2020 11:45:27 GMT" time="3.074" errors="0">
  <testcase name="@BeforeTest setUp" time="0.0" classname="com.zyto.test.base.ZytoBaseTest">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;browser&amp;apos; is required by BeforeTest on method setUp but has not been marked @Optional or defined
in D:\New folder\ZytoTestAutomation\testng.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'browser' is required by BeforeTest on method setUp but has not been marked @Optional or defined
in D:\New folder\ZytoTestAutomation\testng.xml
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase name="@BeforeMethod SetUp" time="2.828" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &amp;apos;3.14.0&amp;apos;, revision: &amp;apos;aacccce0&amp;apos;, time: &amp;apos;2018-08-02T20:19:58.91Z&amp;apos;
System info: host: &amp;apos;LAPTOP-QBC6F12R&amp;apos;, ip: &amp;apos;192.168.1.101&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_172&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-QBC6F12R', ip: '192.168.1.101', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_172'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:573)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
at com.zyto.test.base.ZytoBase.init_remoteWebDriver(ZytoBase.java:99)
at com.zyto.test.base.ZytoBase.init_driver(ZytoBase.java:67)
at com.zyto.test.TestCases.Create_New_Account_Test.SetUp(Create_New_Account_Test.java:31)
Caused by: java.net.ConnectException: Failed to connect to /127.0.0.1:4444
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:242)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:160)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
... 34 more
Caused by: java.net.ConnectException: Connection refused: connect
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
at java.net.PlainSocketImpl.connect(Unknown Source)
at java.net.SocksSocketImpl.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:240)
... 56 more
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase name="@AfterMethod EndTest" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- @AfterMethod EndTest -->
  <testcase name="@BeforeMethod SetUp" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase name="@AfterMethod EndTest" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- @AfterMethod EndTest -->
  <testcase name="@BeforeMethod SetUp" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase name="@AfterMethod EndTest" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- @AfterMethod EndTest -->
  <testcase name="@AfterTest tearDown" time="0.0" classname="com.zyto.test.base.ZytoBaseTest">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="Click_On_Create_button" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- Click_On_Create_button -->
  <testcase name="Click_Signin_button" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- Click_Signin_button -->
  <testcase name="Click_world_icon" time="0.0" classname="com.zyto.test.TestCases.Create_New_Account_Test">
    <skipped/>
  </testcase> <!-- Click_world_icon -->
</testsuite> <!-- Test -->
